src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -249,7 +249,7 @@ class OPENASSETIO_CORE_EXPORT Manager {
    *
    * @param context The calling context.
    *
-   * @return a `TraitsData` for each element in `traitSets`.
+   * @return a `TraitsData` for each element in @p traitSets.
```

Comment:
[question] Is it only the Manager docs that need `@p`-ing?  Or is it just that this change is deliberately limited to Manager docs?  I looked through a few others and didn't see anywhere that needs `@p`s.

Reply:
Ah OK, already answered, https://github.com/OpenAssetIO/OpenAssetIO/pull/935#discussion_r1205769432

> We may have others in other files.

Reply:
I was just trying to avoid scope-creeping this PR. It felt odd to have  a mix of conventions within these files, so hoped sorting out the manager classes to at least be consistent was a reasonable compromise for now?